text-ui-test/input.txt

```diff
@@ -0,0 +1,7 @@
+todo borrow book
+deadline return book /by June 6th
+event project meeting /at Mon 2-4pm
+list
+done 2
+list
```

Comment:
Perhaps you could consider adding test cases designed to throw exceptions, such as done 200 or delete 100 as well